### PR TITLE
feat(ConditionalsEditor): search for plot path

### PR DESCRIPTION
**Pull request by junomaly**
users can now also search for plot path in conditionals editor

### CODE REVIEW

**Review by henbagle — APPROVED** [review]

**Comment by Mgamerz** [comment]
Looks good to me.

File: LegendaryExplorer/LegendaryExplorer/Tools/ConditionalsEditor/ConditionalsEditorWindow.xaml.cs [hunk 1]
```diff
@@ -172,7 +172,8 @@ private void Search()
                 try
                 {
                     string text = entry.Conditional.Decompile();
-                    if (text.Contains(searchText, StringComparison.OrdinalIgnoreCase))
+                    string entryPlotPath = entry.PlotPath;
```
Comment (SirCxyrtyx) [inline LegendaryExplorer/LegendaryExplorer/Tools/ConditionalsEditor/ConditionalsEditorWindow.xaml.cs]: `PlotPath` will be null for any conditional not in the plot database (such as ones from mods), causing an exception to be thrown on the next line.
```suggestion
                    string entryPlotPath = entry.PlotPath ?? "";
```

**Review by SirCxyrtyx — CHANGES_REQUESTED** [review]

**Comment by junomaly** [comment]
Thank you, @SirCxyrtyx

**Comment by SirCxyrtyx** [comment]
Thanks for contributing!

**Pull request merged**